oc_cdtapi/ForemanAPI/dto.py

```
@@ -0,0 +1,24 @@
+from typing import Optional
+
+
+@dataclass
+class HostComputeAttributes:
+    cpus: int
+    memory_mb: Optional[int]
+    disk_size: Optional[int]
+    power_state: Optional[str]
+
+    @classmethod
+    def from_json(cls, data):
```

Ruff (0.15.5)

[warning] Missing return type annotation for classmethod `from_json`

(ANN206)